src/libs/actions/Device/index.ts

```diff
@@ -29,7 +29,7 @@ function getDeviceID() {
 /**
  * Saves a unique deviceID into Onyx.
  */
-function setDeviceID() {
+function setDeviceID(): void {
```

Comment:
```suggestion
function setDeviceID() {
```

We don't need to specify `void` returns